config/scout.php

```diff
@@ -77,8 +78,8 @@
     */
 
     'chunk' => [
-        'searchable' => 500,
-        'unsearchable' => 500,
+        'searchable' => 100,
+        'unsearchable' => 100,
```

Comment:
A chunk size of 500 ultimately results in OOM errors being thrown. Tuning this down to 100 completely resolved the issue for me locally.